src/styles/event-venue/frontend.pcss

```diff
@@ -76,3 +126,80 @@
 		min-height: 270px;
 	}
 }
+
+/* Theme specific overrides */
+
+/* Twenty-twenty-one theme */ 
+.theme-twentytwentyone .tribe-block__venue .tribe-block__venue__meta .tribe-block__venue__website a {
+	color: #334aff;
+}
+
+/* Twenty-twenty theme */
+.theme-twentytwenty .tribe-block__venue .tribe-block__venue__meta .tribe-block__venue__name h3,
+.theme-twentytwenty .tribe-block__venue .tribe-block__venue__meta .tribe-block__venue__address,
+.theme-twentytwenty .tribe-block__venue .tribe-block__venue__meta .tribe-block__venue__address a,
+.theme-twentytwenty .tribe-block__venue .tribe-block__venue__meta .tribe-block__venue__phone,
+.theme-twentytwenty .tribe-block__venue .tribe-block__venue__meta .tribe-block__venue__website {
+	font-size: 1.65rem;
+}
+
+.theme-twentytwenty .tribe-block__venue .tribe-block__venue__meta .tribe-block__venue__website a {
+	color: #334aff;
+}
+
+/* Divi */
+.theme-Divi .tribe-block__venue .tribe-block__venue__meta .tribe-block__venue__name h3,
+.theme-Divi .tribe-block__venue .tribe-block__venue__meta .tribe-block__venue__address,
+.theme-Divi .tribe-block__venue .tribe-block__venue__meta .tribe-block__venue__phone,
+.theme-Divi .tribe-block__venue .tribe-block__venue__meta .tribe-block__venue__website  {
+	padding-bottom: 0;
+}
+
+.theme-Divi .tribe-block__venue .tribe-block__venue__meta .tribe-block__venue__website a {
+	color: #334aff;
+}
+
+/* Genesis */
+.theme-genesis .tribe-block__venue .tribe-block__venue__meta .tribe-block__venue__name h3,
+.theme-genesis .tribe-block__venue .tribe-block__venue__meta .tribe-block__venue__address,
+.theme-genesis .tribe-block__venue .tribe-block__venue__meta .tribe-block__venue__phone,
+.theme-genesis .tribe-block__venue .tribe-block__venue__meta .tribe-block__venue__website   {
+	font-size: 1.65rem;
+}
+
+.theme-genesis .tribe-block__venue .tribe-block__venue__meta .tribe-block__venue__website a {
+	color: #334aff;
+}
+
+/* Avada */
+.theme-Avada .tribe-block__venue .tribe-block__venue__meta .tribe-block__venue__name .fusion-responsive-typography-calculated {
+	font-size: 1.2rem;
+	font-weight: normal;
+	letter-spacing: normal;
+}
+
+.theme-Avada .tribe-block__venue .tribe-block__venue__meta .tribe-block__venue__address a,
+.theme-Avada .tribe-block__venue .tribe-block__venue__meta .tribe-block__venue__website a {
+	color: #334aff;
+}
+
+.theme-Avada .tribe-block__venue .tribe-block__venue__meta {
+	max-width: 80%;
+
+	@media screen and (min-width: 768px) {
+		max-width: 40%;
+	}
+}
+
+.theme-Avada .tribe-block__venue .tribe-block__venue__map {
+
+	@media screen and (min-width: 768px) {
+		max-width: 70%;
+	}
+}
+
+/* Enfold */
+.theme-enfold .tribe-block__venue .tribe-block__venue__meta .tribe-block__venue__name h3,
+.theme-enfold .tribe-block__venue .tribe-block__venue__meta .tribe-block__venue__website a {
+	color: #334aff;
+}
```

Comment:
Let's nest these so we aren't repeating unnecessarily

Reply:
I've added some nesting